### DISCUSSION TITLE
ART 1.10.2

### DISCUSSION THREAD

**Discussion by beat-buesser** [Trusted-AI/adversarial-robustness-toolbox#1711] [Announcements]
This release of ART 1.10.2 provides updates to ART 1.10.

# Added

[None]

# Changed

- Changed `PyTorchClassifier` to use a new optimizer when cloned with `clone_for_refitting` (#1580)
- Changed class names of `art.estimators.gan.*` and `art.estimators.generator.*` to follow naming convention (#1655)
- Changed `Mp3CompressionPyTorch` and `PyTorchDeepSpeech` to add support for samples in 2D non-object arrays (#1680, #1702)
- Changed file name `python_object_detector.py` to `pytorch_object_detector.py` to follow naming convention (#1687)
- Changed `CarliniLInfMethod` by adding argument for `batch_size` (#1699).

# Removed

[None] 

# Fixed

- Fixed required dependency on TensorFlow (#1655)
- Fixed bug in `ImperceptibleASRPyTorch` by adding missing `.detach().cpu()` and `.cpu()` calls (#1677)
- Fixed bug in `art.estimators.certification.randomized_smoothing` estimators to correctly apply Gaussian noise (#1678)
- Fixed bug in `GaussianNoise` the post-processing defence to keep number of dimensions constant during normalisation (#1684)
- Fixed bug in `RobustDPatch` for channels first images to correctly un-transform loss gradients (#1693)
- Fixed bug in support for numpy arrays in logger of `PoisoningAttackCleanLabelBackdoor` (#1698)

<hr /><em>This discussion was created from the release <a href='https://github.com/Trusted-AI/adversarial-robustness-toolbox/releases/tag/1.10.2'>Bump version to ART 1.10.2</a>.</em>
